docs/streams/developer-guide/dsl-api.html

```diff
@@ -3097,152 +3098,994 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
+            if (record.value() == null) {
+                return; // Skip null events
+            }
+
+            // Derive the current day from the event timestamp
+            final long timestamp = System.currentTimeMillis(); // Use system time for simplicity
+            final String currentDay = DATE_FORMATTER.format(Instant.ofEpochMilli(timestamp));
+            // Retrieve the current count for the day
+            Long dailyCount = stateStore.get(currentDay);
+            if (dailyCount == null) {
+                dailyCount = 0L;
+            }
+            // Increment the count
+            dailyCount++;
+            stateStore.put(currentDay, dailyCount);
+
+            // Emit the current day&#39;s count
+            context.forward(record.withValue(String.format(&quot;Day: %s, Car Count: %s&quot;, currentDay, dailyCount)));
+        }
+    }
+}</code></pre>
+                <h5 id="replacing-slang-in-text-messages">Replacing Slang in Text Messages</h5>
+                <ul>
+                    <li><strong>Idea:</strong> A messaging stream contains user-generated content, and you want to replace slang
+                        words
+                        with their formal equivalents (e.g., &quot;u&quot; becomes &quot;you&quot;, &quot;brb&quot; becomes &quot;be
+                        right back&quot;). The operation only modifies the message value and keeps the key intact.</li>
+                    <li><strong>Real-World Context:</strong> In customer support chat systems, normalizing text by replacing slang
+                        with
+                        formal equivalents ensures that automated sentiment analysis tools work accurately and provide reliable
+                        insights.</li>
+                </ul>
+                <pre class="line-numbers"><code class="language-java">package org.apache.kafka.streams.kstream;
+
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.processor.api.ContextualFixedKeyProcessor;
+import org.apache.kafka.streams.processor.api.FixedKeyRecord;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+
+public class ReplacingSlangTextInMessagesExample {
+    private static final Map&lt;String, String&gt; SLANG_DICTIONARY = Map.of(
+            &quot;u&quot;, &quot;you&quot;,
+            &quot;brb&quot;, &quot;be right back&quot;,
+            &quot;omg&quot;, &quot;oh my god&quot;,
+            &quot;btw&quot;, &quot;by the way&quot;
+    );
+    private static final String INPUT_MESSAGES_TOPIC = &quot;input-messages-topic&quot;;
+    private static final String OUTPUT_MESSAGES_TOPIC = &quot;output-messages-topic&quot;;
+
+    public static void replaceWithProcessValues(final StreamsBuilder builder) {
+        KStream&lt;String, String&gt; messageStream = builder.stream(INPUT_MESSAGES_TOPIC);
+        messageStream.processValues(SlangReplacementProcessor::new).to(OUTPUT_MESSAGES_TOPIC);
```

Comment:
nit: don't chain `.to(...);` in the same line, but move to next line -- easier to read.